DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: AsioHeaders
 Type: Package
 Title: 'Asio' C++ Header Files
-Version: 1.22.1-2.1
-Date: 2024-08-02
+Version: 1.28.2-1
+Date: 2025-04-08
```

Comment:
I prefer to leave these changes to the maintainer (me) when PRs / uploads are finalised.  Maybe revert?

Reply:
This is reverted.